### ISSUE TITLE
Two VPCs for one app seems overkill

### ISSUE THREAD

**Issue by neekolas** [prisma/prisma-templates#9]
Between this [Fargate config](https://github.com/prismagraphql/prisma-templates/blob/bcec4a8a830f7ffc5cab9440cf8cbc1dd5293471/aws/fargate.yml#L79) and the [MySQL](https://github.com/prismagraphql/database-templates/blob/master/aws/mysql.yml) template, hosting your own Prisma instance on AWS requires you to create two new VPCs. That seems unnecessary, especially since Amazon caps your account at[ 5 VPCs total per region](https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Appendix_Limits.html#vpc-limits-vpcs-subnets) by default.

The easiest thing would be to add some conditions in to both templates and let you specify an existing VPC and subnet. The better solution IMO is to merge the two templates and put everything into one VPC. It would also cut down on a bunch of copy and paste between the two configs.

I'm happy to handle rejigging the templates and testing it out. Just want to make sure there wasn't a rationale behind the setup.

**Comment by mcmar** [comment]
I think the only rationale is that they're currently 2 separate templates. I would personally love it if you could rejigger the templates to use the same VPC because I've already hit my cap of 5 VPCs in the US-West-2 region.

**Comment by FWirtz** [comment]
I agree, two VPCs really don't seem reasonable - even though you can increase the limit of 5 via a support ticket.

Putting everything into one VPC should be easily doable. You could still completely isolate the DB from Fargate if you place them into different private subnets, though I don't think that's necessary anyways.

**Comment by jordan-erisman** [comment]
Anyone had any luck doing this? I am struggling to get my single file to work properly.

**Comment by flowirtz** [comment]
@erisman20 yeah I have both running in the same VPC. I have a terraform file I could clean and share with you, in case you're interested.

**Comment by tonyfromundefined** [comment]
@erisman20 I created CloudFormation templates with 1 VPC. Check my repository.

https://github.com/tonyfromundefined/prisma-aws-cloudformation
